starter/js/atm.js

```diff
@@ -1,31 +1,64 @@
 //Begin with the document ready function
+$(document).ready(function() {
+ //Checking account deposit function
+ //On click of the depositChecking button
 
-    //Checking account deposit function
+ $("#depositChecking").click(function(event) {
+     console.log(event);
 
-      //On click of the depositChecking button
+     //Get value from the amountChecking input field
+     var depositAmount = parseInt($("#amountChecking").val(), 10);
+     var currentBalance = parseInt($('#checkingBalance').text().replace('$', ''), 10);
+     //Take that value and add it to the existing value in the checkingBalance div
+     var totalBalance = depositAmount + currentBalance;
+     $("#checkingBalance").text("$" + totalBalance);
+ });
 
-        //Get value from the amountChecking input field
 
-        //Take that value and add it to the existing value in the checkingBalance div
+ //Checking account withdrawl funtion
+ //On click of the withdrawChecking button
+ //Get value from the amountChecking input field
 
-    //Checking account withdrawl funtion
+   $("#withdrawChecking").click(function(event) {
+     console.log(event);
 
-      //On click of the withdrawChecking button
+     //Get value from the amountChecking input field
+     var withAmount = parseInt($("#amountChecking").val(), 10);
+     var currentBalance = parseInt($('#checkingBalance').text().replace('$', ''), 10);
+     //Take that value and add it to the existing value in the checkingBalance div
 
-        //Get value from the amountChecking input field
+     var totalBalance = currentBalance - withAmount;
 
-        // If that value is greater than the value in the account ignore that action
-        // In other words if this would put the account into a negative balance do not allow it
+     if (totalBalance >= withAmount) {
```

Comment:
Here `totalBalance` is the difference between your `currentBalance` and `withAmount`. The conditional should check if that difference is greater than __0__, not `if (totalBalance >= withAmount)`.

For example, if I try to deposit $30 and withdraw $25, I can't, because `totalBalance` is 5 and `withAmount` is 25.